ops/storage-ddls/cassandra-setup.cql

```diff
@@ -102,7 +102,6 @@ CREATE TABLE sitesettings(
     cogspeechsvctoken text,
     cogvisionsvctoken text,
     cogtextsvctoken text,
-    featureservicenamespace: text,
```

Comment:
Already include in line 98 above (and this line has a trailing colon character).